src/Talis/Persona/Client/Base.php

```diff
@@ -1,7 +1,10 @@
 <?php
 namespace Talis\Persona\Client;
+require __DIR__ . '/../../../../vendor/autoload.php';
```

Comment:
I don't think you'd want to deal with or refer to the autoloader at all since it's a client library.  Presumably it's being called from something else. We had this same problem in Tripod.  Is your problem that your tests are blowing up because of dependencies?